go/test/endtoend/vtorc/utils/utils.go

```diff
@@ -582,11 +582,29 @@ func RunSQL(t *testing.T, sql string, tablet *cluster.Vttablet, db string) (*sql
 	defer conn.Close()
 
 	// RunSQL
-	return execute(t, conn, sql)
+	return RunSQLWithSuperReadOnly(t, sql, tablet, db, false)
 }
 
-func execute(t *testing.T, conn *mysql.Conn, query string) (*sqltypes.Result, error) {
+// RunSQLWithSuperReadOnly is used to run a SQL statement on the given tablet with super read only set to true or false
+func RunSQLWithSuperReadOnly(t *testing.T, sql string, tablet *cluster.Vttablet, db string, withSuperReadOnly bool) (*sqltypes.Result, error) {
+	// Get Connection
+	tabletParams := getMysqlConnParam(tablet, db)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	conn, err := mysql.Connect(ctx, &tabletParams)
+	require.Nil(t, err)
+	defer conn.Close()
+
+	// RunSQL
+	return execute(t, conn, sql, withSuperReadOnly)
+}
```

Comment:
This is another case where it feels like we're avoiding testing the fact that Vitess works properly when super_read_only is enabled. No?

Reply:
if you closely look at I have introduced a flag 'withSuperReadOnly' ... during test setup we introduce errantGTID (https://github.com/planetscale/vitess/pull/77/files#diff-9162b364b991658ff1575922d4a008aecc15e44fd27e64678f24cce330033d9fR108) thats the only place where we pass true to this flag. For all other places we keep it false. Hence the tests are running with super-read-only false.